modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/ToAkkaConverters.scala

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.bluebrain.nexus.rdf.akka
+
+import akka.http.scaladsl.model.Uri
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+import ch.epfl.bluebrain.nexus.rdf.Node
+import ch.epfl.bluebrain.nexus.rdf.Node.IriNode
+
+import scala.util.Try
+
+/**
+  * Conversions from rdf data types and Akka [[akka.http.scaladsl.model.Uri]].
+  */
+trait ToAkkaConverters {
+
+  /**
+    * Attempts to convert argument [[Node]] to Akka [[Uri]].
+    */
+  def asAkka(node: Node): Either[String, Uri] = node match {
+    case IriNode(iri) => asAkka(iri)
+    case other        => Left(s"${other.show} cannot be converted to URI.")
+  }
+
+  /**
+    * Attempts to convert argument [[AbsoluteIri]] to Akka [[Uri]].
+    */
+  def asAkka(iri: AbsoluteIri): Either[String, Uri] =
```

Comment:
not sure we need to wrap this in a `Try`, @umbreak what do you think?

Reply:
yes, if we use `asUri` we don't need `Try`